pkg/services/sqlstore/migrator/mysql_dialect.go

```diff
@@ -138,7 +138,9 @@ func (db *Mysql) CleanDB() error {
 	return nil
 }
 
-func (db *Mysql) TruncateDB() error {
+// TruncateDBTables truncates all the tables
```

Comment:
Should be periods at the end of this and the following sentence (and in the other occurrences)